_overviews/contributors/index.md

```diff
@@ -0,0 +1,809 @@
+(written in sbt’s syntax):
+
+~~~ scala
+"org.slf4j" % "slf4j-simple" % "1.7.25"
+~~~
+
+Its `groupId` is `org.slf4j`, its `artifactId` is `slf4j-simple`, and its `version` is `1.7.25`.
+
+To make your library usable by other people, we recommend publishing it to the Maven Central repository.
+This process requires having a [Sonatype](https://central.sonatype.org/) account and a PGP key pair to
+sign the binaries.
+
+### Create a Sonatype Account and Project
+
+Follow the instructions given on the [OSSRH Guide](https://central.sonatype.org/pages/ossrh-guide.html#initial-setup)
+to create a new Sonatype account (unless you already have one) and to create a new project ticket. This latter
+step is where you define the `groupId` that you will release to. You can use a domain name that you already own,
+otherwise a common practice is to use `io.github.(username)` (where `(username)` is replaced with your GitHub
+username).
+
+This step has to be performed only once per `groupId` you want to have.
+
+### Create a PGP Key Pair
+
+Sonatype [requires](https://central.sonatype.org/pages/requirements.html) that you sign the published files
+with PGP. Follow the instructions [here](https://central.sonatype.org/pages/working-with-pgp-signatures.html)
+to generate a key pair and to distribute your public key to a key server.
+
+This step has to be performed only once per person.
+
+### Setup Your Project
+
+In case you use sbt, we recommend using the [sbt-sonatype](https://github.com/xerial/sbt-sonatype) 
+and [sbt-pgp](https://www.scala-sbt.org/sbt-pgp/) plugins to publish your artifacts. Add the following
+dependencies to your `project/plugins.sbt` file:
+
+~~~ scala
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.4")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
+~~~
+
+And make sure your build fulfills the [Sonatype requirements](https://central.sonatype.org/pages/requirements.html)
+by defining the following settings:
+
+~~~ scala
+// used as `artifactId`
+name := "library-example"
+
+// used as `groupId`
+organization := "ch.epfl.scala"
+
+// open source licenses that apply to the project
+licenses := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
+
+description := "A library that does nothing useful"
+
+import xerial.sbt.Sonatype._
+sonatypeProjectHosting := Some(GitHubHosting("scalacenter", "library-example", "julien.richard-foy@epfl.ch"))
+
+// publish to the sonatype repository
+publishTo := sonatypePublishTo.value
+~~~
+
+Put your Sonatype credentials in a `$HOME/.sbt/1.0/sonatype.sbt` file:
+
+~~~ scala
+credentials += Credentials("Sonatype Nexus Repository Manager",
+        "oss.sonatype.org",
+        "(Sonatype user name)",
+        "(Sonatype password)")
+~~~
+
+(Put your actual user name and password in place of `(Sonatype user name)` and `(Sonatype password)`)
+
+**Never** include this file within your project.
```

Comment:
maybe "check this file into version control" instead of "include this file within your project"